src/test/resources/io/valkey/springframework/data/valkey/config/namespace.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+		xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		xmlns:valkey="http://www.springframework.org/schema/valkey"
+		xmlns:task="http://www.springframework.org/schema/task"
+		xsi:schemaLocation="http://www.springframework.org/schema/beans https://www.springframework.org/schema/beans/spring-beans.xsd
+				http://www.springframework.org/schema/task https://www.springframework.org/schema/task/spring-task.xsd
+				http://www.springframework.org/schema/valkey https://www.springframework.org/schema/valkey/spring-valkey-1.0.xsd">
```

Comment:
Until we publish the schema to a specific location some tooling, such as IDEs, will complain that this file cannot be downloaded.  The tests still run since we have a schema mapping at `src/main/resources/META-INF/spring.schemas`.